DigitalLearningSolutions.Web/Scripts/trackingSystem/emailDelegates.ts

```diff
@@ -8,18 +8,26 @@ const route = `TrackingSystem/Delegates/Email/AllEmailDelegateItems?${queryParam
 // eslint-disable-next-line no-new
 new SearchSortFilterAndPaginate(route, false, false, true, 'EmailDelegateFilter');
 
+function alertResultCount(): void {
+  const resultCount = document.getElementById('results-count') as HTMLSpanElement;
+  resultCount.setAttribute('role', '');
+  resultCount.setAttribute('role', 'alert');
+}
+
 function selectAll(): void {
   const allCheckboxes = document.querySelectorAll('.delegate-checkbox') as NodeListOf<HTMLInputElement>;
   allCheckboxes.forEach((checkbox) => {
     if (!checkbox.checked) checkbox.click();
   });
+  alertResultCount();

```

Comment:
I think these alert calls need to go in the event listener declaration. It's not a part of the select all, it belongs to the button press i.e. the button action becomes "Select all then alert the user"
I've moved the selectAll functions to a common place as part of 441, and not all pages will necessarily have the results count element to be able to do this and will need an alternative way to announce it if we want this functionality.